test/uk/gov/hmrc/transitmovementsconverter/models/ModelImplicitsSpec.scala

```diff
@@ -64,8 +64,12 @@ class ModelImplicitsSpec extends AnyFreeSpec with ScalaFutures with Matchers wit
     }
 
     // In the stock XSDs, this incorrectly tried to read " NCTS5.1", not "NCTS5.1", hence this test.
+    /*
+    case object NCTS5u461 extends PhaseIDtype { override def toString = " NCTS5.1" }
+    case object NCTS5u460 extends PhaseIDtype { override def toString = "NCTS5.0" }
+     */
     "A phase ID of 'NCTS5.1' should be accepted" in {
-      ModelImplicits.phaseIDtypeReads.reads(JsString("NCTS5.1")) mustBe JsSuccess(NCTS5u461)
+      ModelImplicits.phaseIDtypeReads.reads(JsString(" NCTS5.1")) mustBe JsSuccess(NCTS5u461)
```

Comment:
These tests should not have been changed -- it was intentional that these were here so that the errant leading space didn't need to be included. The comment I wrote infers this, and I'm fairly sure I added code somewhere to ensure that this test passes.

This test should be reverted to ensure that reading "NCTS5.1" gives NCTS5u461 -- and if it fails, we should fix it so the test passes, not change the test so the code passes.

Reply:
I think the XSDs have added some space in the new version - will  provide more details in our chat

Reply:
...my apologies, that was also me. I removed the space in the old version as it was causing issues with the Json stuff. From our chat:

> in the old version, that was also an issue, so the simplest fix I had was to just remove that space in the xsd
> when validating, the XML validators ignore that space (for some reason), but the Json validators do not.

Apologies, I should have made that clearer somewhere!

Reply:
updated stypes.xsd to remove empty space